GS01SUMO/GS01SUMO_R00004_ingest.csv

```diff
@@ -0,0 +1,80 @@
+mi.dataset.driver.ctdmo_ghqr.imodem.ctdmo_ghqr_imodem_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00004/cg_data/dcl11/imm/ctdmo01/*.DAT,GS01SUMO-RII11-02-CTDMOQ011,recovered_host,Available,
+mi.dataset.driver.ctdmo_ghqr.sio.ctdmo_ghqr_ct_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00004/instruments/dcl11/imm/CTDMO_sn37-12586/*.hex,GS01SUMO-RII11-02-CTDMOQ011,recovered_inst,Available,
+mi.dataset.driver.ctdmo_ghqr.imodem.ctdmo_ghqr_imodem_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00004/cg_data/dcl11/imm/ctdmo02/*.DAT,GS01SUMO-RII11-02-CTDMOQ012,recovered_host,Available,
+mi.dataset.driver.ctdmo_ghqr.sio.ctdmo_ghqr_ct_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00004/instruments/dcl11/imm/CTDMO_sn37-12589/*.hex,GS01SUMO-RII11-02-CTDMOQ012,recovered_inst,Available,
+mi.dataset.driver.ctdmo_ghqr.imodem.ctdmo_ghqr_imodem_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00004/cg_data/dcl11/imm/ctdmo03/*.DAT,GS01SUMO-RII11-02-CTDMOQ013,recovered_host,Available,
+mi.dataset.driver.ctdmo_ghqr.sio.ctdmo_ghqr_ct_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00004/instruments/dcl11/imm/CTDMO_sn37-12940/*.hex,GS01SUMO-RII11-02-CTDMOQ013,recovered_inst,Available,
+mi.dataset.driver.ctdmo_ghqr.imodem.ctdmo_ghqr_imodem_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00004/cg_data/dcl11/imm/ctdmo04/*.DAT,GS01SUMO-RII11-02-CTDMOQ014,recovered_host,Available,
+mi.dataset.driver.ctdmo_ghqr.sio.ctdmo_ghqr_ct_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00004/instruments/dcl11/imm/CTDMO_sn37-13556/*.hex,GS01SUMO-RII11-02-CTDMOQ014,recovered_inst,Available,
+mi.dataset.driver.ctdmo_ghqr.imodem.ctdmo_ghqr_imodem_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00004/cg_data/dcl11/imm/ctdmo05/*.DAT,GS01SUMO-RII11-02-CTDMOQ015,recovered_host,Available,
+mi.dataset.driver.ctdmo_ghqr.sio.ctdmo_ghqr_ct_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00004/instruments/dcl11/imm/CTDMO_sn37-13557/*.hex,GS01SUMO-RII11-02-CTDMOQ015,recovered_inst,Available,
+mi.dataset.driver.ctdmo_ghqr.imodem.ctdmo_ghqr_imodem_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00004/cg_data/dcl11/imm/ctdmo06/*.DAT,GS01SUMO-RII11-02-CTDMOQ016,recovered_host,Available,
+mi.dataset.driver.ctdmo_ghqr.sio.ctdmo_ghqr_ct_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00004/instruments/dcl11/imm/CTDMO_sn37-13577/*.hex,GS01SUMO-RII11-02-CTDMOQ016,recovered_inst,Available,
+mi.dataset.driver.ctdmo_ghqr.imodem.ctdmo_ghqr_imodem_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00004/cg_data/dcl11/imm/ctdmo07/*.DAT,GS01SUMO-RII11-02-CTDMOQ017,recovered_host,Available,
+mi.dataset.driver.ctdmo_ghqr.sio.ctdmo_ghqr_ct_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00004/instruments/dcl11/imm/CTDMO_sn37-13588/*.hex,GS01SUMO-RII11-02-CTDMOQ017,recovered_inst,Available,
+#mi.dataset.driver.ctdmo_ghqr.imodem.ctdmo_ghqr_imodem_recovered_driver,,GS01SUMO-RII11-02-CTDMOQ031,recovered_host,Not Deployed,Not deployed

```

Comment:
CTDMOQ031 and CTDMOQ033 are not part of the design.  They were only deployed for D00001.  I don't understand why they are included here.

Reply:
Agreed, I generated this ingest csv by copying GS01SUMO D00003, which had them included and commented out. Since you agree, I can delete those lines and push a new copy.